IFC4x3/Sections/Core data schemas/Schemas/IfcProductExtension/Entities/IfcObservation/Documentation.md

```diff
@@ -0,0 +1 @@
+A generic representation of an observation that can be assigned to another product using _IfcRelAssignsToProduct_
```

Comment:
Documentation should include a hint on how the actual observation data is linked with this entity.
Period missing at the end of sentence.

Reply:
Tried to add another sentence explaining this. @SergejMuhic added the "." before.